Challenge Problems - GSSoC/pair_sum_problem_BST.cpp

```diff
@@ -10,4 +10,66 @@ The following tasks need to be done for an entire score:
 2. Space taken should be less than O(n).
 3. Explanations for all methods and time/space complexities.
 
-*/
\ No newline at end of file
+*/
+
+#include <vector>
+ #include <algorithm>
+ 
+ class BinaryTreeNode {
+ public :
+ T data;
```

Comment:
Use proper indentation in the complete code.

Reply:
can u please explain me where proper indentation needs to be done........

Reply:
@CoderJolly I guess he's referring to one tab that you should have inserted in the next line after `public:` since it started a new block.

Reply:
ok, i will do it.....